grep-000x-upstreaming.md

```diff
@@ -0,0 +1,140 @@
+# GREP [XXXX] -- Upstreaming Process
+
+
+- Original Author: Marcus Müller <mmueller@gnuradio.org>
+- Champion: Marcus Müller <mmueller@gnuradio.org>
+- Status: Draft
+
+History:
+- 15-Feb-2017: Initial Draft
+- 05-Mar-2018: Conversion to GREP process
+
+## Abstract
+
+This GREP+ strives to establish a guideline for
+
+* when to submit blocks for becoming part of the upstream GNU Radio tree and
+* how to ensure maintenance and maintainability
+
+## Copyright / License
+
+[WTFPL](http://www.wtfpl.net/txt/copying/) Version 2 or later
```

Comment:
Every open source lawyer will tell you this is not a license. Please choose something more approriate.

Reply:
https://www.gnu.org/licenses/license-list.html#WTFPL it's not recommended by the FSF, but recognized.